README.md

```diff
@@ -105,6 +105,25 @@ The Home app offers no extra buttons for the Quiet and Powerful Modes. All setti
 
 - If the plugin affects the general responsiveness and reliability of your Homebridge setup, you can run it as an isolated [child bridge](https://github.com/homebridge/homebridge/wiki/Child-Bridges).
 
+## Latest Beta
+
+Beta versions of this plugin are made available from time to time for those who wish to test new features. Please be aware that there may still be issues that need addressing (see 'Contributing' below for reporting any issues you find). The current beta release version, along with its changes, are listed below. To install the latest beta on Homebridge, click on the three vertical dots to the right of 'Settings' for your current Homebridge Panaonic AC Platform and choose 'Install Alternate Version'.
```

Comment:
I'd keep the general information and manage the version-specific bits in GitHub Issues or a separate file for release notes, but don't feel hindered to go with any approach you prefer.